components/sidebar/channel_create.jsx

```diff
@@ -102,7 +96,7 @@ export default class ChannelCreate extends React.PureComponent {
     };
 
     renderDirect = () => {
-        const ariaLabelDM = `${Utils.localizeMessage('sidebar.createDirectMessage', 'Create new direct message')} ${Utils.localizeMessage('accessibility.button.button', 'button')}`;
+        const ariaLabelDM = `${Utils.localizeMessage('sidebar.createDirectMessage', 'Create new direct message').toLowerCase()}`;
```

Comment:
Don't really need the template literal string here anymore.

```suggestion
        const ariaLabelDM = Utils.localizeMessage('sidebar.createDirectMessage', 'Create new direct message').toLowerCase();
```